dpnp/tests/test_linalg.py

```diff
@@ -1456,12 +1454,12 @@ def test_different_paths(self, dtype):
         # Use einsum to compare to not have difference due to sum round-offs:
         result1 = dpnp.einsum(",i->", s_dp, a_dp)
         result2 = dpnp.einsum("i->", s_dp * a_dp)
-        assert_array_equal(result1.asnumpy(), result2.asnumpy())
+        assert_array_equal(result1, result2)
```

Comment:
both inputs seems dpnp.ndarray's, should we use `assert dpnp.allclose(...)` instead?